packages/builder/src/utils/wagmi.ts

```diff
@@ -118,7 +114,7 @@ function enabledChainsAndProviders() {
 
       chains.push(chain);
     });
-  } else if (appEnv === "production") {
+  } else if (config.env === "production") {
```

Comment:
Does this refer to `NODE_ENV` or the deployment environment? It might be useful to disambiguate. The indexer uses `config.deploymentEnvironment` IIRC to avoid confusion.

Reply:
I updated it with config.appEnv to keep it similar to the previous one